app/routers/dayview.py

```diff
@@ -211,10 +212,8 @@ async def dayview(
     current_time_with_attrs = CurrentTimeAttributes(date=day)
     inter_day = international_days.get_international_day_per_day(session, day)
     tasks = (
-        session.query(Task)
-        .filter(Task.owner_id == user.user_id)
-        .filter(Task.date == day.date())
-        .order_by(Task.time)
+        session.query(Task).filter(Task.owner_id == user.user_id).filter(
+            Task.date == day.date()).order_by(Task.time)
```

Comment:
Please revert back to older form

Reply:
it get error in flake8 in ther older form